config/settings.yml

```diff
@@ -163,6 +163,7 @@ feature:
   show_reconciliation_deposit_number: <%= ENV.fetch("RECONCILIATION_DEPOSIT_NUMBER", false) %>
   show_daily_rate_option: true
   sanger_enabled_service: <%= ENV.fetch("SANGER_ENABLED_SERVICE", false) %>
+  alternative_csv_format: <%= ENV.fetch("ALTERNATIVE_CSV_FORMAT", false) %>
```

Comment:
nit: maybe we can try using a different name that explains where it's used? Something like `well_plate_alternative_csv_format`?